### PR TITLE
Add logging verbosity flag backed by klog

### PR DESCRIPTION
**Pull request by ironcladlou**
Add a `-v` logging verbosity flag and switch to klog for consistency
with upstream and because zapr doesn't implement V levels directly anyway.

### CODE REVIEW

File: log/log.go [hunk 1]
```diff
@@ -2,19 +2,14 @@ package log
...
 var Logger logr.Logger
 
 func init() {
-	// Set up logging.
-	zapLogger, err := zap.NewDevelopment(zap.AddCallerSkip(1), zap.AddStacktrace(zap.FatalLevel))
-	if err != nil {
-		panic(err)
-	}
-	Logger = zapr.NewLogger(zapLogger).WithName("router")
+	Logger = klogr.New()
```
Comment (frobware) [inline log/log.go]: Do we need the `init()`? Can't this be:

    var Logger logr.Logger = klogr.New()

In fact, do we need the package? Can this just move to `main()`?

Comment (frobware) [reply log/log.go]: > In fact, do we need the package? Can this just move to `main()`?

Oh, unless you want this as `log.Logger` - which does read a lot better.

Comment (ironcladlou) [reply log/log.go]: init can be removed, but the package is because otherwise all the library code would have to depend on the command package.

Comment (ironcladlou) [reply log/log.go]: Fixed

**Pull request merged**